config/settings.yml

```diff
@@ -2,8 +2,12 @@
 date_format_str: '%Y-%m-%d %H:%M:%S.%L'
 rolling_indexer:
   batch_size: 500
-  pause_time_between_docs: .2
-  pause_time_between_batches: 0
+  # in seconds
+  pause_time_between_docs: 0.2
+  # commitWithin does a soft commit by default, so a little more than commitWithin is desired (seconds)
```

Comment:
If `commitWithin` is 1s and we want a value that is "a little more," I wonder if it is worth testing other values here to see the effect on 1) duplicates included across batches, and 2) throughput. E.g., the old value of 5s and perhaps 2s. Also happy to ship this PR and have these tweaks be follow-up work if someone is inclined. 🤷🏻‍♂️

Reply:
Yes - and I was experimenting last night, and I will do more today.  (It's in the google doc).  1.2 seconds was too short.  :-)

Reply:
Do we want a value a little more than `commitWithin` or a little more than `autoSoftCommit/maxTime` (from the Solr config)?